github4s/jvm/src/test/scala/github4s/unit/DecodersSpec.scala

```diff
@@ -50,4 +51,12 @@ class DecodersSpec extends FlatSpec with Matchers with FakeResponses {
     decode[CombinedStatus](emptyListResponse) should be('left)
   }
 
+  "NonEmptyList Decoder" should "return a valid NonEmptyList for a valid JSON" in {
+    decode[NonEmptyList[Int]]("[1,2,3]") should be(Right(NonEmptyList.of(1, 2, 3)))
+  }
+
+  it should "return an error for an empty list" in {
+    decode[NonEmptyList[Int]](emptyListResponse) should be('left)
+  }
+
```

Comment:
Hmm this seems to be a bit dodgy decoder, converting both `A` and `[A]`, but as that seems what is needed, I'll add that test :+1: